docs/src/examples/BayesianNODE_NUTS.md

```diff
@@ -0,0 +1,277 @@
+# Bayesian Neural ODEs: NUTS
+The DiffEqFlux.jl library in Julia can be seamlessly combined with Bayesian estimation
+librarries likes AdvancedHMC.jl and Turing.jl. This enables converting Neural ODEs
```

Comment:
```suggestion
libraries likes AdvancedHMC.jl and Turing.jl. This enables converting Neural ODEs
```

Reply:
I think we need a more convincing introduction, this doesn't show the motivation of why we want to integrate bayesian with neural odes, just throw some of the abstract from the paper here

Reply:
My thinking was that when a person comes to visit this tutorial,  having seen the paper, that person would be convinced of why this integration is needed.

However, doesn't hurt to be more comprehensive. 

I have tried to make it comprehensive now.